Package.swift

```diff
@@ -33,10 +33,14 @@ let package = Package(
         .library(
             name: "SwaggerServiceModel",
             targets: ["SwaggerServiceModel"]),
+        .library(
+            name: "OpenAPIServiceModel",
+            targets: ["OpenAPIServiceModel"]),
     ],
     dependencies: [
-        .package(url: "https://github.com/tachyonics/SwaggerParser.git", from: "0.6.4"), 
+        .package(url: "https://github.com/tachyonics/SwaggerParser.git", from: "0.6.4"),
         .package(url: "https://github.com/jpsim/Yams.git", from: "4.0.0"),
+        .package(url: "https://github.com/Josh-Gearou/OpenAPIKit.git", from: "2.4.0"),
```

Comment:
Is there a reason we are depending on Josh's fork of this dependency?

Reply:
No reason other than it looks like we forked SwaggerParser.  Also I assumed you did that so we could control changes we may want to make on our side.

Reply:
Is there a reason why the forked SwaggerParser would work as a dependency but the forked OpenAPIKit would not?